packages/rum-core/src/domain/rumEventsCollection/view/trackViewMetrics.ts

```diff
@@ -105,14 +107,20 @@ export function trackViewMetrics(
     stopCLSTracking = noop
   }
 
+  const { stop: stopINPTracking, getInteractionToNextPaint } = trackInteractionToNextPaint(loadingType, lifeCycle)
+
   return {
     stop: () => {
       stopLoadingTimeTracking()
       stopCLSTracking()
       stopScrollMetricsTracking()
+      stopINPTracking()
     },
     setLoadEvent,
-    viewMetrics,
+    getViewMetrics: () => {
+      viewMetrics.interactionToNextPaint = getInteractionToNextPaint()
+      return viewMetrics
+    },
```

Comment:
`getInteractionToNextPaint` does not really recompute the INP. INP is computed each time a [PerformanceEventTiming](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEventTiming) is collected.

`getInteractionToNextPaint` only serve for the case no INP has been computed but performance.interactionCount > 0, meaning that some interaction did happen. It can occur because of the performanceObserver 40ms threshold. 
In this scenario we need to report an INP at 0 instead of no INP for the page.